api/src/trace.rs

```diff
@@ -63,3 +64,72 @@ pub enum ExecutionEvent {
         msg: String,
     },
 }
+
+impl From<ExecutionTrace> for InvocationTrace {
+    fn from(e_trace: ExecutionTrace) -> InvocationTrace {
+        let mut invocation_stack: Vec<InvocationTrace> = Vec::new();
+
+        for event in e_trace.clone().events.into_iter() {
+            match event {
+                ExecutionEvent::GasCharge { .. } | ExecutionEvent::Log { .. } => {}
+                ExecutionEvent::Call { from, to, method, params, value } => {
+                    invocation_stack.push(InvocationTrace {
+                        from: Address::new_id(from),
```

Comment:
I think we could and should update InvocationTrace to use `from: ActorID` too.

Reply:
https://github.com/filecoin-project/builtin-actors/issues/1365

Will have to update vm_api in builtin's first then will followup here.